### ISSUE TITLE
Ground Items Customization GUI Keybinding

### ISSUE THREAD

**Issue by KoryMac** [runelite/runelite#7516]
Currently on runelite pressing ALT shows an in-game gui for customizing what items on the ground are shown, not shown, or highlighted if the Ground Items plugin is enabled. However, I and many other players use ALT to control our cameras. This means that whenever we try to turn our cameras a bunch of clutter pops up onto the screen and I personally often end up highlighting an item that I had hidden by clicking to turn my camera right after pressing alt, and accidentally clicking on an item name that just popped up after I pressed alt. This presents a bit of a nuisance. I know that most people use their middle mouse button to turn the camera, however my mouse does not have a middle mouse button. Furthermore, when playing on a laptop using the trackpad, there is no middle mouse button available.

I would like to request that an addition to the Ground Items plugin's settings menu be added. This would be a line where you can set a custom keybind for toggling the ingame customization GUI. I would like to be able to set this functionality to my `/~ key, to my caps lock, or to specifically my right alt and not my left alt.

If adding the option to customize this keybind is not possible, then I request that a toggle be added to only use the right alt on the keyboard, and ignore the left alt. 

Thank you for reading! I know this post might be a bit wordy, but I decided to follow the template instead of quickly summarizing =D.

**Comment by deathbeam** [comment]
Duplicate of #7831 (easier to read)
